### PR TITLE
Register Sedara schema extension

### PR DESCRIPTION
**Pull request by cabrel**
#### Description of changes:

Registering a schema extension id for Sedara.

### CODE REVIEW

**Comment by floydtree** [comment]
@cabrel  One quick note, can you add an entry in the `unreleased` section in CHANGELOG.md file? You can refer to the previous entries for reference.

**Comment by mikeradka** [comment]
LGTM once the `CHANGELOG` is updated

**Comment by pagbabian-splunk** [comment]
Same: LGTM but needs `CHANGELOG`

**Review by floydtree — APPROVED** [review]

**Review by mikeradka — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -36,6 +36,7 @@ Thankyou! -->
 ### Breaking changes
 
 ### Misc
+1. New Extension registration for Sedara. #951
```
Comment (Aniak5) [inline CHANGELOG.md:39]: Should this go under the `Platform Extensions` category under `Added` ?

Comment (floydtree) [reply CHANGELOG.md:39]: It's a vendor extension right? Platform extension would something like - Windows, MacOS, Android etc

Comment (Aniak5) [reply CHANGELOG.md:39]: Oooh, good point.

**Review by Aniak5 — APPROVED** [review]
LGTM

**Pull request merged**